src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final int PRECISION = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private double getRandomPropertyValue() {
+        return (double) random.nextInt(MAX_VALUE) / PRECISION;
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE.name(), 10);
```

Comment:
Move this values to constants:
```suggestion
        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
```